log/log.go

```diff
@@ -96,6 +97,22 @@ func JSONLog(b bool) {
 	initLogging()
 }
 
+// FileLog turns on logging to file
+func FileLog(filepath string) {
```

Comment:
there is probably no need to print output to stdout if it is logged to a file. and we should expose config options for file rotations